code/modules/mob/living/carbon/xenomorph/xeno_helpers.dm

```diff
@@ -2,10 +2,14 @@
 
 
 /mob/living/carbon/Xenomorph/can_ventcrawl()
-	return (mob_size != MOB_SIZE_BIG)
+	if(mob_size == MOB_SIZE_BIG)
+		return FALSE
+	if(isXenoWarrior(src) || isXenoDefender(src))
```

Comment:
Good idea. A more efficient way to do this would be by:
```
/mob/living/carbon/Xenomorph/Warrior/can_ventcrawl()
	return FALSE

/mob/living/carbon/Xenomorph/Defender/can_ventcrawl()
	return FALSE
```